.github/workflows/pkgci.yml

```diff
@@ -35,7 +35,12 @@ jobs:
     uses: ./.github/workflows/pkgci_regression_test_cpu.yml
     needs: [build_packages]
 
-  regression_test_amdgpu:
-    name: Regression Test AMDGPU
-    uses: ./.github/workflows/pkgci_regression_test_amdgpu.yml
+  regression_test_amdgpu_vulkan:
+    name: Regression Test AMDGPU-Vulkan
+    uses: ./.github/workflows/pkgci_regression_test_amdgpu_vulkan.yml
+    needs: [build_packages]
+
+  regression_test_amdgpu_rocm:
+    name: Regression Test AMDGPU-ROCm
+    uses: ./.github/workflows/pkgci_regression_test_amdgpu_rocm.yml
     needs: [build_packages]
```

Comment:
It looks like this is running on presubmit for all changes, and the job is not labeled as experimental (while the code it exercises is). Is this workflow / job required to pass for PRs to be merged?

I'd prefer we find a way to make those distinctions clear before adding tests for experimental code.

[`ci.yml`](https://github.com/openxla/iree/blob/main/.github/workflows/ci.yml), for example, has some jobs labeled as postsubmit only, but you can still opt-in to running them when either directly via the PR description or indirectly via files touched: https://github.com/openxla/iree/blob/main/docs/developers/developing_iree/contributing.md#ci-behavior-manipulation, https://github.com/openxla/iree/blob/dc9d600ddc626a0b2fc8f4802836db5ea6058b73/build_tools/github_actions/configure_ci.py#L115-L135

Reply:
We're going to run all the tests we have all the time if we care about the platform. None of these pkgci tests are blocking anything right now. I'll do a round of normalizing names and organizing before that changes. For now, having the signal and progress is important to me.

Reply:
More tests are always welcome :D

The detail I want to figure out is how we can add these tests in a way that is most useful to those that care about them without putting excess burden on the rest of the project. It sounds like the ROCm code is relatively stable but it is using some older interfaces, and the CI/runners are relatively new. Putting "experimental" in the CI job name and ensuring that it doesn't block PRs may be enough for now.

Reply:
It is a small team and we can communicate as such. If we need to disable this or break, we can/will.